modules/openedx/mongodb/cluster.tf

```diff
@@ -17,8 +17,7 @@ resource "mongodbatlas_cluster" "cluster" {
   disk_size_gb                = local.disk_size_gb
   provider_disk_iops          = local.disk_iops
   provider_volume_type        = local.volume_type
-  provider_backup_enabled     = var.backup_enabled
+  cloud_backup                = var.backup_enabled
```

Comment:
When I applied the changes, terraform complained about `provider_backup_enabled` being a deprecated variable (though that is not on the docs yet).

Since I was upgrading to `0.13` I also took the opportunity to remove some deprecated settings.